### PR TITLE
Feature page 5

### PR DESCRIPTION
**Pull request by bauHau**
This feature adds the page 5 to the application.

### CODE REVIEW

File: page_5.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+			<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+			tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+			quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+			consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+			cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+			proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+		</header>
+
+		<section>
+			<h2>Section one</h2>
```
Comment (fullStackDevs) [inline page_5.html]: The content of this section should be set inside a container.

File: page_5.html [hunk 2]
```diff
@@ -25,6 +25,12 @@ <h1 class="text-center">Page 5</h1>
 			<p>Modification done by the fullStackDevs</p>
 			<p>Modification done by bauHau</a>
 			<p>Some stuff</p>
+			<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
```
Comment (fullStackDevs) [inline page_5.html:28]: This should be in italic.

**Review by fullStackDevs — CHANGES_REQUESTED** [review]
Please make the changes mentioned in this review.

File: page_5.html (hunk 2 above)
Comment (bauHau) [reply page_5.html:28]: The client specification changed in the last moment and now it's not required to have the font italic.

**Review by fullStackDevs — APPROVED** [review]
This looks great

**Pull request merged**